flows/routers/tests/tests.go

```diff
@@ -810,44 +802,79 @@ func hasOnlyPhraseTest(origHays []string, hays []string, pins []string) XTestRes
 // Numerical Test Functions
 //------------------------------------------------------------------------------------------
 
-var parseableNumberRegex = regexp.MustCompile(`^[$£€]?([\d,][\d,\.]*([\.,]\d+)?)\D*$`)
-
+var altNumerals = map[rune]rune{
+	'٠': '0', // Arabic
```

Comment:
the neat thing about Arabic numerals is that it's basically the same but with different digit characters (i.e. decimal and left-to-right)